### PR TITLE
refactored import_commands function

### PR DESCRIPTION
**Pull request by gurbirsandha**
all telecommands should now get added into database

### CODE REVIEW

**Comment by gurbirsandha** [comment]
This should be good to go except for a couple of minor merge conflicts.

File: groundstation/backend_api/utils.py [hunk 1]
```diff
@@ -30,13 +30,13 @@ def decorate(*args, **kwargs):
 
     return decorate
 
-def add_telecommand(command_name, num_arguments, is_dangerous):
+def add_telecommand(command_name, num_arguments, is_dangerous, about_info):
```
Comment (jmmabanta) [inline groundstation/backend_api/utils.py:33]:
```suggestion
def add_telecommand(command_name, num_arguments, is_dangerous, about_info=''):
```
Add a blank default about_info so unit tests pass

File: groundstation/static/js/components/Help.jsx [hunk 2]
```diff
@@ -5,212 +5,381 @@ import Button from '@material-ui/core/Button';
...
+          <Typography
+            variant="body1"
+            style={{
+              paddingLeft: '20px',
+              paddingRight: '20px',
+              paddingBottom: '10px'
+            }}
+          >
+            Send telecommands to the satellite via the live commands page.
+            Commands are formatted as "prefix.command_name(arg1 arg2 ...) "
```
Comment (jmmabanta) [inline groundstation/static/js/components/Help.jsx:232]:
```suggestion
            Commands are formatted as "prefix.command_name(arg1,arg2,arg3,...) "
```
The command format has changed where args are now separated by a comma

**Review by jmmabanta — CHANGES_REQUESTED** [review]
I really like the look of the new help screen for the commands! Just need to provide a blank default value to the `about_info` parameter in `add_telecommand()` so unit tests pass and a small change in the help menu about how args are formatted (which are now separated by a comma).

I also fixed the merge conflict so make sure you `git pull` first so you get these changes locally.

**Review by jmmabanta — APPROVED** [review]

**Pull request merged**